src/pages/components/modal/usage.mdx

```diff
@@ -222,29 +217,35 @@ Labels or text of any kind should never be in the modal margins.
 
 ## Content
 
-### Main elements
+In IBM Style, the modal is referred to as a dialog. 
+
+### Title
+
+- Use a brief verb phrase that describes the modal’s task or purpose.  
+- If the modal is accessed by clicking a button, use the button label for the modal title:
+    - For example, if the user clicks the **Create access group** button to open the modal, _Create access group_ is the modal title. Modals might also be accessed by clicking a menu item or an icon.
+    - Be aware of situations where there’s a tendency to mix and match button or icon labels and modal titles. For example, if the icon name is **New user**, use _New user_ as the modal title, not _Create user_. 
+- If your product has a more conversational tone, use articles (a, an, the) in the modal title even though the button label doesn't include the article, for example _Create an access group_.
+- Sometimes, you might need to provide contextual information for the modal, such as when the actions performed using the modal apply to a specific entity and you want to make it clear which entity the actions apply to. In this case, you can use the optional label that is displayed above the title. For example, if the user needs to know the path to the object that they are editing, you might put the path in the optional label. [
```

Comment:
Entity is just a thing. It could be any thing. I changed "entity" to "item" - hopefully that makes more sense. I also used sub-bullets as recommended. Thanks!